eclair-core/src/test/scala/fr/acinq/eclair/payment/PaymentInitiatorSpec.scala

```diff
@@ -69,6 +70,19 @@ class PaymentInitiatorSpec extends TestKit(ActorSystem("test")) with fixture.Fun
     withFixture(test.toNoArgTest(FixtureParam(nodeParams, initiator, payFsm, multiPartPayFsm, sender, eventListener)))
   }
 
+  test("send TLV payment with custom TLV records to node") { f =>
```

Comment:
nit: would better fit with the rest of the file:
```suggestion
  test("forward payment with user custom tlv records") { f =>
```